app/renderer/js/main.js

```diff
@@ -280,6 +282,9 @@ class ServerManagerView {
 			if (this.activeTabIndex === index) {
 				return;
 			} else if (hideOldTab) {
+				if (this.tabs[this.activeTabIndex].props.role === 'function' && this.tabs[this.activeTabIndex].props.name === 'Settings') {
```

Comment:
Can you document a bit here and in `menu.js` about the code? Other than this it's good to merge after squashing and making that checkbox change.